app/services/external_api/webex_service.rb

```diff
@@ -16,8 +16,8 @@ def create_conference(virtual_hearing)
     body = {
       "jwt": {
         "sub": virtual_hearing.subject_for_conference,
-        "Nbf": virtual_hearing.hearing.scheduled_for.beginning_of_day.to_i,
-        "Exp": virtual_hearing.hearing.scheduled_for.end_of_day.to_i
+        "Nbf": virtual_hearing.nbf,
+        "Exp": virtual_hearing.exp
```

Comment:
I don't think the keys are case-sensitive, but I'm hesitant to take any chances.

```suggestion
        "nbf": virtual_hearing.nbf,
        "exp": virtual_hearing.exp
```

Reply:
I know I'm scared to touch the keys but I will change it if you want me to!

Reply:
I'll test it on my GFE.

Reply:
The capital letters didn't make a difference. You can disregard the original comment.